playwright/tests/integration/data-classes.spec.ts

```diff
@@ -55,3 +55,20 @@ test('data class tags', async ({ page }) => {
   await overview.hasCardWithTag('BusinessData', 'Business Data');
   await overview.hasCardWithTag('EntitySample', 'Entity');
 });
+
+test('data classes graph', async ({ page }) => {
+  const { neo, overview } = await openDataClasses(page);
+  await overview.hasGroup(`Project: ${TEST_PROJECT}`);
+  await overview.viewToggle.locator('button').nth(1).click();
```

Comment:
if we change the order of this buttons, this will fail. Why not call the button by it's name? 
`getByRole('button', {name: 'Graph View'});`